### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by ericdallo** [turtlequeue/setup-babashka#3]
Could you confirm if this would be enough to use with Windows?

```bash
iwr -useb get.scoop.sh | iex
scoop bucket add scoop-clojure https://github.com/littleli/scoop-clojure
scoop bucket add extras
scoop install babashka --independent
echo "$HOME\scoop\shims" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
```

from [here](https://github.com/clj-commons/rewrite-clj/blob/f640a38d9267adb0a6a7e06453a13402aa61c9b1/.github/workflows/unit-test.yml#L66-L70)

**Comment by ericdallo** [comment]
ping @nha

**Comment by nha** [comment]
Looks promising! I will have a look @ericdallo

**Comment by nha** [comment]
Looks like it should work @ericdallo  ! I am not a scoop/windows user though: do you know if there is a way to specify a version?

```
scoop install babashka vX.X.X --independent
```
also do you know what `--independent` does here? I could not find any doc

**Comment by ericdallo** [comment]
I'm not a windows user as well 😆, maybe @HUMORCE can help with those questions?

**Comment by borkdude** [comment]
In some projects I just use these powershell snippets to download and unzip bb.exe:

https://github.com/babashka/pod-babashka-buddy/blob/37ce1185d43e5f5855e56425ee46550a167687c0/appveyor.yml#L48-L50

And then I just call it with `bb.exe` in the local directory, for the CI build.

**Comment by nha** [comment]
Seems even simpler without scoop. I'll go with that for now. Thanks !

**Comment by borkdude** [comment]
The above may not be enough to get bb.exe on the path though, not sure if that is a requirement for this tool.

**Comment by nha** [comment]
Yes, I just assume the CI tests running `bb -version` will be enough to fail if that's not the case.

**Comment by nha** [comment]
Just released v1.2.0 with windows support @ericdallo
